typhos/panel.py

```diff
@@ -266,6 +266,36 @@ def _create_row_label(self, attr, dotted_name, tooltip, long_name=None):
             label.setToolTip(_tooltip)
         return label
 
+    def _get_long_name(self, device, attr, dotted_name) -> str:
+        """
+        Check the signal for its long_name, if it exists. 
+        Until Ophyd makes it a standard signal, need to manually check
+        the device and its components for the name.
+
+        Parameters
+        -----------
+            device: (any)
+                The Ophyd.Device with component signals
+            attr: (str)
+                The str name of the signal attribute
+            dotted_name: (str)
+                The full dotted name to the signal
+
+        Returns
+        --------
+            str or None
+        """
+        long_name = None
+        try:
+            if hasattr(getattr(device, attr), 'long_name'):
+                long_name = getattr(device, attr).long_name
+        except AttributeError:
+            # Then maybe we have a nested component and can't touch the signal
+            if hasattr(getattr(device, dotted_name), 'long_name'):
+                long_name = getattr(device, dotted_name).long_name
+        return long_name
```

Comment:
Super petty optional don't-allocate-the-variable edit
```suggestion
        try:
            if hasattr(getattr(device, attr), 'long_name'):
                return getattr(device, attr).long_name
        except AttributeError:
            # Then maybe we have a nested component and can't touch the signal
            if hasattr(getattr(device, dotted_name), 'long_name'):
                return getattr(device, dotted_name).long_name
```